lib/router.js

```diff
@@ -100,12 +101,18 @@ Router.prototype._lookup = function route(path, node) {
         if (node.value && node.value.security) {
             permissions = permissions.concat(node.value.security);
         }
+        if (node.value && node.value.filters) {
```

Comment:
Lots of repeated checks for node.value. Might be worth wrapping that in a single if(node.value) { ... }